Assets/UI/Panels/productionpanel_CQUI.lua

```diff
@@ -214,22 +215,29 @@ function GetData()
     local buildQueue = pSelectedCity:GetBuildQueue();
 
     for row in GameInfo.Units() do
-        if row.MustPurchase and buildQueue:CanProduce( row.Hash, true ) and row.PurchaseYield == "YIELD_FAITH" then
+        if row.MustPurchase and buildQueue:CanProduce( row.Hash, true ) then
             local isCanProduceExclusion, results     = buildQueue:CanProduce( row.Hash, false, true );
             -- If a unit is purchase only, then "isDisabled" needs to be True, as that disables the button control that
             -- allows the religious units to be enqueued
-            -- TODO: It'd be nice to remove the unnecessary part about the production cost from the Tooltip string
             local isDisabled  :boolean = row.MustPurchase -- not isCanProduceExclusion;
             local sAllReasons :string = ComposeFailureReasonStrings( isDisabled, results );
             local sToolTip    :string = ToolTipHelper.GetUnitToolTip( row.Hash, MilitaryFormationTypes.STANDARD_MILITARY_FORMATION, buildQueue ) .. sAllReasons;
+            -- Remove the part about the production cost from the tooltip
+            local prodPartPattern = "%[NEWLINE%]%[NEWLINE%]Cost:.+%[ICON_Production%] Production"
```

Comment:
What happens with other languages than English?

Reply:
so excited about figuring out the pattern matching with lua that I forgot about the localization.